.flowconfig

```diff
@@ -10,8 +10,6 @@ flow/
 src/core_modules/capture-core/flow/
 
 [options]
-suppress_comment= \\(.\\|\n\\)*\\$FlowSuppress
```

Comment:
looks like according to this we dont need those https://github.com/facebook/flow/issues/8403#issuecomment-644942810